client/src/components/PreviousTripsCard/PreviousTripsCard.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import PreviousTripsCard from "./PreviousTripsCard";
+
+storiesOf("PreviousTripsCard", module)
+  .add("Default", () => (
+    <PreviousTripsCard
+      trips={true}
+      link="https://www.figma.com/file/ggst6OoJWkwaV2DGq4nBzA/CoTripper?node-id=0%3A1"
```

Comment:
Instead of linking to an actual site that wouldn't be used in the actual app, link to `"/"` or `"#"`.